db/seed.sql

```diff
@@ -4476,7 +4473,7 @@ CREATE TABLE `eav_entity_attribute` (
+INSERT INTO `eav_entity_attribute` VALUES (1,1,1,1,1,10),(2,1,1,1,2,20),(3,1,1,1,3,20),(4,1,1,1,4,30),(5,1,1,1,5,40),(6,1,1,1,6,50),(7,1,1,1,7,60),(8,1,1,1,8,70),(9,1,1,1,9,80),(10,1,1,1,10,25),(11,1,1,1,11,90),(12,1,1,1,12,81),(13,1,1,1,13,115),(14,1,1,1,14,120),(15,1,1,1,15,82),(16,1,1,1,16,83),(17,1,1,1,17,100),(18,1,1,1,18,85),(19,1,1,1,19,86),(20,1,1,1,20,110),(21,1,1,1,21,121),(22,2,2,2,22,10),(23,2,2,2,23,20),(24,2,2,2,24,30),(25,2,2,2,25,40),(26,2,2,2,26,50),(27,2,2,2,27,60),(28,2,2,2,28,70),(29,2,2,2,29,80),(30,2,2,2,30,90),(31,2,2,2,31,100),(32,2,2,2,32,100),(33,2,2,2,33,110),(34,2,2,2,34,120),(35,2,2,2,35,130),(36,2,2,2,36,131),(37,2,2,2,37,132),(38,2,2,2,38,133),(39,2,2,2,39,134),(40,2,2,2,40,135),(41,1,1,1,41,87),(42,3,3,4,42,1),(43,3,3,4,43,2),(44,3,3,4,44,4),(45,3,3,4,45,5),(46,3,3,4,46,6),(47,3,3,4,47,7),(48,3,3,4,48,8),(49,3,3,5,49,10),(50,3,3,5,50,20),(51,3,3,5,51,30),(52,3,3,4,52,12),(53,3,3,4,53,13),(54,3,3,4,54,14),(55,3,3,4,55,15),(56,3,3,4,56,16),(57,3,3,6,57,10),(58,3,3,6,58,30),(59,3,3,6,59,40),(60,3,3,6,60,50),(61,3,3,6,61,60),(62,3,3,4,62,24),(63,3,3,4,63,25),(64,3,3,5,64,40),(65,3,3,5,65,50),(66,3,3,4,66,10),(67,3,3,6,67,5),(68,3,3,6,68,6),(69,3,3,5,69,51),(87,4,4,7,88,6),(93,4,4,8,95,8),(102,4,4,7,104,14),(103,4,4,7,105,15),(104,4,4,7,106,16),(105,4,4,7,107,17),(106,4,4,7,108,18),(107,4,4,7,109,19),(108,4,4,7,110,20),(111,3,3,4,113,3),(112,3,3,4,114,17),(114,4,4,7,116,11),(118,4,4,7,120,102),(119,4,4,7,121,103),(121,4,4,7,123,104),(122,4,4,7,124,105),(123,4,4,7,125,106),(124,4,4,7,126,107),(125,4,4,7,127,108),(130,4,4,7,70,2),(132,4,4,7,71,3),(134,4,4,7,72,10),(136,4,4,7,74,4),(138,4,4,7,79,8),(140,4,4,7,84,6),(142,4,4,7,94,11),(144,4,4,7,102,9),(146,4,4,7,112,7),(148,4,4,7,119,12),(150,4,4,7,128,1),(152,4,4,7,129,5),(154,4,4,7,131,13),(156,4,4,7,132,14),(158,4,4,10,85,1),(160,4,4,10,86,2),(162,4,4,10,87,3),(164,4,4,10,93,4),(166,4,4,9,81,2),(168,4,4,9,82,3),(170,4,4,9,83,4),(172,4,4,9,115,1),(174,4,4,8,75,1),(176,4,4,8,76,2),(178,4,4,8,77,3),(180,4,4,8,78,4),(182,4,4,8,89,5),(184,4,4,8,117,6),(186,4,4,8,118,7),(188,4,4,8,122,8),(190,4,4,11,97,1),(192,4,4,11,98,2),(194,4,4,11,99,3),(196,4,4,11,100,4),(198,4,4,11,101,5),(200,4,4,11,103,6),(202,4,4,12,73,1),(204,4,4,12,91,3),(206,4,4,12,92,4),(208,4,4,12,96,2),(210,4,4,12,111,5),(212,4,4,12,130,6);
```

Comment:
Somewhere in here are the assignments of the `author` and `cover` attributes to the attribute set (entity).